src/Data/Dao/PiantaDao.php

```diff
@@ -16,12 +22,14 @@ class PiantaDao
 
     public function __construct()
     {
-      $Db = new Database();
-      $this->connection = $Db->connect();
+      $db = new Database();
+      $this->connection = $db->connect();
       $this->I_PIANTA = "INSERT INTO pianta(genere, specie, nome, co2, descrizione) VALUES(:genere, :specie, :nome, :co2, :descrizione)";
       $this->D_PLANT = "DELETE FROM pianta WHERE id=:id";
       $this->U_QR_PLANT = "UPDATE pianta SET qrcode = :qrcode WHERE id = :id";
-      $this->S_PLANT = "SELECT * FROM pianta AS a INNER JOIN stato_pianta AS b ON a.id = b.id_pianta WHERE a.id = :id ;";
+      $this->S_PLANT = "SELECT * FROM pianta WHERE id = :id ;";
+      $this->S_PLANT_STATUS = "SELECT * FROM stato_pianta WHERE id_pianta = :id_pianta ;";
```

Comment:
@shewraar sarebbe meglio mettere la query che fa la select nella tabella `stato_pianta` in un Dao apposito visto che é un 'altra tabella nel database, e prevedere all'interno del model `Pianta` un metodo che permetta di caricare la parte dello stato